sql/hive/src/main/scala/org/apache/spark/sql/hive/TableReader.scala

```diff
@@ -175,7 +175,8 @@ class HadoopTableReader(
             def updateExistPathSetByPathPattern(pathPatternStr: String): Unit = {
               val pathPattern = new Path(pathPatternStr)
               val fs = pathPattern.getFileSystem(hadoopConf)
-              val matches = fs.globStatus(pathPattern)
+              val (matches, timeCost) = Utils.timeTakenMs(fs.globStatus(pathPattern))
+              logInfo(s"Get FileStatus for path '$pathPatternStr' costs $timeCost ms.")
```

Comment:
logDebug?

Reply:
Personally, I prefer to use logInfo, because 

1) the message is not noisy, `pathPatternStr` matches the table path, which means if the query access 10 tables, just generate 10 messages. 

2) the PR itself aims to give more information when the user's job is stuck on listing files, the most important information is `pathPatternStr`, and I suppose most users will not turn on debug level log on production, then users have less opportunity to this message.

Reply:
OK, I don't feel super strongly about it.

Reply:
@srowen Do you have any concerns before merging it?

Reply:
I'm neutral, would not merge it but do not object to it

Reply:
emm